src/agentlab/llm/huggingface_utils.py

```diff
@@ -11,7 +11,7 @@
 from agentlab.llm.prompt_templates import PromptTemplate, get_prompt_template
 
 
-class HFBaseChatModel:
+class HFBaseChatModel:  # we need this to be a subclass of AbstractChatModel
```

Comment:
we might need to change the structure of those files a bit, we need to import AbstractChatModel here but that would make a circular import
it feels relevant to move every ChatModel in chat_api.py to this file and renaming it to something appropriate.
it would also feel relevant to rename chat_api.py to something more in line with ModelArgs i thimk ?

Reply:
I just had a quick look, I agree I think moving huggingface's chat model classes to chat_api.py would be cleaner. We could keep only static functions in hugginface_utils.py to not clutter chat_api.py too much